internal/client/k8s/metrics.go

```diff
@@ -35,14 +34,13 @@ func (c *Client) ListCapsuleMetrics(ctx context.Context) (iterator.Iterator[*cap
 			}
 
 			for _, ms := range ml.Items {
-				cid, err := uuid.Parse(capsuleIDFromLabels(ms.GetLabels()))
-				if err != nil {
-					p.Error(err)
-					return
+				cid := capsuleIDFromLabels(ms.GetLabels())
```

Comment:
Tror måske ikke vi har brug for at gemme det her i labels længere?

Reply:
We used to need this because we needed to be able to get the capsule ID from a pod/deployment in kubernetes AFAIR.

Reply:
How should we get the capsule name? It's not a property of the metric, and we don't want to collect for pods that don't belong to a capsule